src/query/test/executor/transform.go

```diff
@@ -48,6 +49,11 @@ type SinkNode struct {
 	Metas  []block.SeriesMeta
 }
 
+// Params returns the Params associated with this node.
```

Comment:
It's a bit inconsistent with commenting on interface methods; comments missing on a few of these `Params()` methods

I'd err on the side of generally removing them unless the particular `Params` method is doing something tricky and needs further explanation